CMakeLists.txt

```diff
@@ -58,21 +58,12 @@ add_executable(glslls
     ${SOURCES}
 )
 
-if (MSVC)
-    target_compile_options(glslls PRIVATE /W4)
-else()
-    target_compile_options(glslls PRIVATE -Wall -Wextra -Wpedantic)
-endif()
```

Comment:
Do we actually want to remove this?

Reply:
That's a good point. We probably want to keep that.

Reply:
If you put this back in in (a separate commit, preferably), I'll merge your stuff.

Reply:
I'm making the changes myself.